crates/config/src/frontmatter.rs

```diff
@@ -21,12 +21,16 @@ pub struct Frontmatter {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub categories: Option<Vec<liquid_core::model::KString>>,
     #[serde(skip_serializing_if = "Option::is_none")]
+    pub authors: Option<Vec<liquid_core::model::KString>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub tags: Option<Vec<liquid_core::model::KString>>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub excerpt_separator: Option<liquid_core::model::KString>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub published_date: Option<DateTime>,
     #[serde(skip_serializing_if = "Option::is_none")]
+    pub updated_date: Option<DateTime>,
```

Comment:
Could you split this out into its own issue / PR?

#197 exists but that is more about automatically inferring it